clients/vault/src/main.rs

```diff
@@ -147,6 +147,7 @@ async fn start() -> Result<(), ServiceError<Error>> {
 
 #[tokio::main]
 async fn main() {
+	console_subscriber::init();
```

Comment:
To [use the `tokio-console`](https://github.com/tokio-rs/console?tab=readme-ov-file#instrumenting-your-program).